mcs/swagger.yml

```diff
@@ -305,4 +349,48 @@ definitions:
         type: integer
         format: int64
         title: total number of groups
-
+  statement:
+    type: object
+    properties:
+      Effect:
+        type: string
+      Action:
+        type: array
+        items:
+          type: string
+      Resource:
+        type: array
+        items:
+          type: string
+  policy:
+    type: object
+    properties:
+      Name:
+        type: string
+      Version:
+        type: string
+      Statement:
```

Comment:
probably do this in lower case? I'm not sure if in upper case makes any difference

Reply:
agree, the json would have Upper case fields

Reply:
i need it to do it in uppercase otherwise `json.Unmarshal` will not be able to parse the properties correctly

Reply:
fixed